submissions/supert111/html-css-popup/index.html

```diff
@@ -14,134 +14,136 @@
+                                            <p>Photo</p>
+                                        </a>
+                                    </li>
+                                    <li class="popupItem">
+                                        <a class="linkFokusPopupItem" href="">
+                                            <img class="popupImg" src="images/apps.png" alt="icon-apps" width="40">
+                                            <p>Apps</p>
+                                        </a>
+                                    </li>
+                                    <li class="popupItem">
+                                        <a class="linkFokusPopupItem" href=""> 
+                                            <img class="popupImg" src="images/documents.png" alt="icon-documents" width="40">
+                                            <p>Document</p>
+                                        </a>
+                                    </li>
+                                </ul>
+                                <input class="checkInput" type="checkbox" id="checkMoreInputId">
+                                <label class="checkMore" for="checkMoreInputId">More</label>
+                                <div class="noCheckItemBlock">
+                                    <ul class="popupList">
+                                        <li class="popupItem noCheckItem">
+                                            <a class="linkFokusPopupItem" href=""> 
+                                                <img class="popupImg" src="images/chat.png" alt="icon-chat" width="40">
+                                                <p>Chat</p>
+                                            </a>
+                                        </li>
+                                        <li class="popupItem noCheckItem">
+                                            <a class="linkFokusPopupItem" href=""> 
+                                                <img class="popupImg" src="images/classroom.png" alt="icon-classroom" width="40">
+                                                <p>Classroom</p>
+                                            </a>
+                                        </li>
+                                        <li class="popupItem noCheckItem">
+                                            <a class="linkFokusPopupItem" href=""> 
+                                                <img class="popupImg" src="images/mars.png" alt="icon-mars" width="40">
+                                                <p>Mars</p>
+                                            </a>
+                                        </li>
+                                        <li class="popupItem noCheckItem">
+                                            <a class="linkFokusPopupItem" href=""> 
+                                                <img class="popupImg" src="images/collection.png" alt="icon-collection" width="40">
```

Comment:
You have text inside your links + icons are decorative images so `alt` attribute should be empty